views/account/profile/main.js

```diff
@@ -41,12 +41,16 @@ function add_row(table, label, value) {
   table.appendChild(tr);
```

Comment:
I didn't get to review the last PR but the changes you made to the profile page are really good! I definitely like the idea of updating values immediately as opposed to when the user hits 'submit.'

The alerts disappear a little quickly and sometimes stack in odd ways, but those are just little things we can fix in the consistency inspection. Awesome job!